gulpfile.js

```diff
@@ -44,19 +44,32 @@ gulp.task("watch-specs", function (done) {
 });
 
 gulp.task("lint", function (done) {
-  var runExternalModule = require("./buildprocess/runExternalModule");
-
-  runExternalModule("eslint/bin/eslint.js", [
-    "lib",
-    "test",
-    "--ext",
-    ".jsx,.js,.ts,.tsx",
-    "--ignore-pattern",
-    "lib/ThirdParty",
-    "--max-warnings",
-    "0"
-  ]);
-
+  const spawnSync = require("child_process").spawnSync;
+  // If upgrading to ESLint 9 without changing config file format:
+  // process.env.ESLINT_USE_FLAT_CONFIG = "false";
+  const result = spawnSync(
+    "node_modules/eslint/bin/eslint.js",
+    [
```

Comment:
This change might not always work, especially when terriajs is inside terriamap yarn workspace folder `terriamap/packags/terriajs` and yarn installs eslint in `terriamap/node_modules` instead of `terriajs/node_modules`

Ideally we keep the original code with a minor modification to link to the correct bin path which is no longer exported.
```
 gulp.task("lint", function (done) {
   var runExternalModule = require("./buildprocess/runExternalModule");
+  var path = require("path");

-  runExternalModule("eslint/bin/eslint.js", [
+  const eslintDir = path.dirname(require.resolve("eslint/package.json"));
+  const eslintExecutable = path.join(eslintDir, "bin", "eslint.js");
+  runExternalModule(eslintExecutable, [
     "lib",
     "test",
     "--ext",
```

Reply:
Fixed, and there's no way I would have figured that out on my own, thanks!